staff/views.py

```diff
@@ -258,6 +275,46 @@ def my_account_staff(request):
 class ClassroomDetailView(DetailView):
     model = Classroom
 
+    def get_forms(self):
+        change_teacher_form = ChangeTeacherForm()
+        add_volunteers_form = AddVolunteersForm()
+        add_students_form = AddStudentsForm()
+        return [change_teacher_form, add_volunteers_form, add_students_form]
```

Comment:
Why not have this return the dict that's used to update the context in `get_context_data`?

```python
            return {
                "change_teacher_form": ChangeTeacherForm(),
                "add_volunteers_form": AddVolunteersForm(),
                "add_students_form": AddStudentsForm(),
            }
```

Using position in a list for different forms is a bit error-prone and can cause refactoring challenges

Reply:
Yes, done